Assets/BossRoom/Scripts/Client/UI/Lobby/LobbyUIMediator.cs

```diff
@@ -72,6 +76,7 @@ public async void CreateLobbyRequest(string lobbyName, bool isPrivate, int maxPl
 
             BlockUIWhileLoadingIsInProgress();
 
+            await m_AuthenticationServiceFacade.ReloginIfNecessary();
```

Comment:
should we do something here to handle when it fails to relogin? like returning to main menu? Or is it something that is already handled by the callers of these methods?

Reply:
So what would happen currently - if such failure occurs, it'll be an unhandled exception here, and it will also show up as a popup message. You're thinking that we should not do the calls after this one, if it fails?

Reply:
If I'm understanding it correctly, if it fails, the other calls we will try to do in the same method would also fail, wouldn't they? If that's the case, then yeah maybe we shouldn't do them to avoid getting spammed with exceptions. Unless this is something that should not happen normally anyways and should be considered an invalid state from which we should not try to recover?